CMS/urls.py

```diff
@@ -9,16 +9,23 @@
 from . import welsh_urls
 
 from courses import urls as courses_urls
+from home import views as home_views
 from institutions import urls as institution_urls
 
 from core import views as core_views
 from search import views as search_views
 from coursefinder import views as coursefinder_views
 
 urlpatterns = [
+    url(r'^widget/', include('widget.urls')),
```

Comment:
I think Django now recommends using path or re_path instead of url. Just a comment though Matt; probably not a good idea to change at this stage.